### PR TITLE
add log for username

### PR DESCRIPTION
**Pull request by tomscut**

### CODE REVIEW

File: hadoop-common-project/hadoop-auth/src/main/java/org/apache/hadoop/security/authentication/server/PseudoAuthenticationHandler.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ private String getUserName(HttpServletRequest request) throws AuthenticationExce
         }
         String[] usernameAndPassword = value.split("-_-");
         if (usernameAndPassword.length != 2) {
-          throw new AuthenticationException("Invalid param of user.name");
+          throw new AuthenticationException("Invalid param of user.name：" + value);
```
Comment (liuhongtong) [inline hadoop-common-project/hadoop-auth/src/main/java/org/apache/hadoop/security/authentication/server/PseudoAuthenticationHandler.java:187]: @tomscut hint the pattern of user.name?

**Comment by liuhongtong** [comment]
Please comment the referenced commit from hadoop project or the issue from hadoop JIRA.

**Comment by tomscut** [comment]
https://github.com/apache/hadoop/commit/136c6f6f7d06

Reference to this commit.

**Comment by tomscut** [comment]
> Please comment the referenced commit from hadoop project or the issue from hadoop JIRA.

Reference to this commit:
https://github.com/apache/hadoop/commit/136c6f6f7d06

File: hadoop-common-project/hadoop-auth/src/main/java/org/apache/hadoop/security/authentication/server/PseudoAuthenticationHandler.java (hunk 1 above)
Comment (tomscut) [reply hadoop-common-project/hadoop-auth/src/main/java/org/apache/hadoop/security/authentication/server/PseudoAuthenticationHandler.java:187]: good idea!

**Comment by liuhongtong** [comment]
LGTM

**Pull request merged**